pkg/api/types.go

```diff
@@ -138,6 +138,21 @@ type ObjectMeta struct {
 	// objects.  Annotation keys have the same formatting restrictions as Label keys. See the
 	// comments on Labels for details.
 	Annotations map[string]string `json:"annotations,omitempty"`
+
+	// List of objects depended by this object. If ALL objects in the list have
+	// been deleted, this object will be garbage collected.
+	OwnerReferences []ObjectReference `json:"ownerReferences,omitempty"`
```

Comment:
@derekwaynecarr: "I wonder if this should be GuardianReferences so it aligns with dependent/guardian relationships. Maybe it's too close to tax season." (https://github.com/kubernetes/kubernetes/commit/135fb5eda98f60b8773405233db9a3e9f5b6898a#commitcomment-17170872)